about.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
- [ ] Everything is working fine, there are not any errors in code and your design👍. I think you forgot to add the `README.md` file. Please kindly fix this issue 😊🍀